react/tic-tac-toe/src/app/screens/Game/components/Square/styles.module.scss

```diff
@@ -1,16 +1,16 @@
 @import 'src/scss/colors.scss';
-
+@import 'src/scss/fonts';
 $square-background: $white;
 $square-border: $dust-gray;
 
 .square {
   background: $square-background;
   border: 1px solid $square-border;
   float: left;
-  font-size: 24px;
+  font-size: $text-big;
   font-weight: bold;
-  line-height: 34px;
   height: 34px;
+  line-height: 34px;
```

Comment:
Try to use relative values for the `line-height`. For example, if the font-size is `16px` and you want the line height to be `32px` do not use `line-height: 32px`, use `line-height: 2` instead. I mean, you can use it, but what if the `font-size` of a certain element changes?